.mage/proto.go

```diff
@@ -69,14 +69,14 @@ func makeProtoc() (func(...string) error, *protocContext, error) {
 	return sh.RunCmd("docker", "run",
```

Comment:
Just define
```go
mountWD := filepath.ToSlash(filepath.Join(filepath.Dir(wd), "lorawan-stack"))
```
above and use it for clarity

`filepath.ToSlash` to support e.g. Windows.